File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/factories/MultiFactoryTest.java
package simulacionBandaMusica.factories;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.models.MultiInstrumentista;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class MultiFactoryTest {

    @Test
    void createMultiTest() {
        MultiInstrumentista multi = MultiFactory.create();
        // MultiInstrumentista no nulo, para ver si se han asignado correctamente los datos
        assertNotNull(multi);
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/factories/MultiFactoryTest.kt
package simulacionBandaMusica.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import simulacionBandaMusica.models.Musico

internal class MultiFactoryTest {
    @Test
    fun createMultiTest() {
        val multi: Musico.MultiInstrumentista = MultiFactory.create()
        // MultiInstrumentista no nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(multi)
    }
}